api-specificatie/openapi.yaml

```diff
@@ -1589,15 +1638,14 @@ components:
           type: array
           items:
             $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/schemas/HalLink'
-          minItems: 1
-
-    Pand_embedded:
-      type: object
-      properties:
-        adressen:
+        adresseerbareObjecten:
           type: array
           items:
-            $ref: '#/components/schemas/AdresHalBasis'
+            $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/schemas/HalLink'
+        nummeraanduidingen:
```

Comment:
Dit moet adressen zijn. Een pand heeft geen relatie met een nummeraanduiding.

Reply:
Adressen staan er ook bij. 

Dit is een functionele discussie die met Cathy gevoerd moet worden. Als er een functionele behoefte is om op basis van een nummeraanduding de bijbehorende panden op te halen kan je toch via alle verblijfsobjecten waar deze nummeraanduiding een hoofdadres of nevenadres van is alle bijbehorende panden ophalen.

Dat is precies hetzelfde als je doet met de adressen. Ik zie in het model dat er geen direcet relatei is, maar dat die via het verblijfsobject loopt. Dan kan je die er toch bij zoeken als provider ?